src/codemagic/tools/firebase_app_distribution/resource_printer.py

```diff
@@ -0,0 +1,34 @@
+from __future__ import annotations
+
+import json
+from typing import Callable
+from typing import Sequence
+
+from codemagic.cli import Colors
+from codemagic.firebase.resources.resource import Resource
+
+
+class ResourcePrinter:
+    def __init__(self, print_json: bool, print_function: Callable[[str], None]):
+        self.print_json = print_json
+        self.print = print_function
+
+    def print_resources(self, resources: Sequence[Resource], should_print: bool):
+        if should_print is not True:
```

Comment:
That can be simplified.
```suggestion
        if not should_print:
```